### ISSUE TITLE
Add a Sass-specific property-no-unknown

### ISSUE THREAD

**Issue by nex3** [stylelint-scss/stylelint-scss#490]
It would be useful to have a version of stylelint's [property-no-unknown](https://stylelint.io/user-guide/rules/property-no-unknown) rule that's aware of Sass's [nested properties](https://sass-lang.com/documentation/style-rules/declarations#nesting), and so can accurately recognize that

```scss
a {
  font: {weight: bold}
}
```

is valid but

```scss
a {
  font: {stuff: bold}
}
```

is invalid.

**Comment by kristerkari** [comment]
Sounds like a good idea for a new SCSS specific rule
